compose/foundation/foundation/src/nativeMain/kotlin/androidx/compose/foundation/Clickable.native.kt

```diff
@@ -29,5 +34,9 @@ internal actual val TapIndicationDelay: Long = 0L
  * Whether the specified [KeyEvent] represents a user intent to perform a click.
  * (eg. When you press Enter on a focused button, it should perform a click).
  */
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
Better to check the native key here (something like `event.skikoEvent.keyCode`). Because `Key` is experimental and in `ui` module. That can lead to backward incompatibility in the future, if users use different versions of `ui` and `foundation`